src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier extends ColorSupplier {
+    public static final int NUMBER_OF_FIGURES_NAME = 5;
+    private static final Random random = new Random();
+    private static final String DEFAULTCOLOR = "white";
+    private static final String DEFAULTNAME = "circle";
+    private static final int DEFAULTRADIUS = 10;

```

Comment:
```suggestion
    private static final int DEFAULT_RADIUS = 10;
```

read about constant naming